src/mistral_common/protocol/instruct/normalize.py

```diff
@@ -90,18 +91,48 @@ def _normalize_json_content(self, content: Optional[str]) -> str:
             normalized_content = content
         return normalized_content
 
-    def _aggregate_content_chunks(self, content: Union[str, List[TextChunk]], chunk_join_str: str = "\n\n") -> str:
+    @overload
+    def _aggregate_content_chunks(
+        self, content: List[Union[str, TextChunk, ThinkChunk]], chunk_join_str: str = "\n\n"
```

Comment:
```suggestion
        self, content: List[Union[str, TextChunk, ThinkChunk]]) -> ...
        chunk_join_str = CHUNK_JOIN_STR
```

`chunk_join_str` seems to be always the same. Can we maybe it a constant variable instead of allowing the user to pass it?